short-survey-checklist.md

```diff
@@ -0,0 +1,21 @@
+# Checklist for MDN short surveys
+
+This document lists the standard operating procedure for creating MDN short surveys. 
+
+# Mandatory fields
+- [ ] Topic: The topic of the survey
+- [ ] Category: E.g. CSS, JavaScript, Accessibility, documentation
+- [ ] Date: When the survey should ideally be run
+- [ ] Objective: What is the goal?
+- [ ] Justification: How is it relevant/interesting to the MDN audience?
+- [ ] Actions: Actions which will be taken based on the results
+- [ ] Audience percentage: Suggested percentage of the MDN audience to be exposed to the survey. Default is 5%, if higher a clear motivation is needed.
+- [ ] Questions: Suggested questions and answers
+- [ ] Other: Any other information which is relevant.
+
+# Frequently requested information
+- [ ] How should the questions be presented? Multiple choice, expanded list, etc
+- [ ] Does any of the lists need to appear in random order?
+- [ ] Do any of the items in a list need to appear in a specific order?
+- [ ] Is there a free form field that needs to be excluded from random order?
+- [ ] Is there a gating question that should diqualify people from taking the survey?
```

Comment:
The MDN short survey guidelines also specifically asked not to ask PII beyond what's included in the privacy policy for the survey takers